server/test/controllers/admin/AdminProgramStatusesControllerTest.java

```diff
@@ -51,13 +52,28 @@ public class AdminProgramStatusesControllerTest extends ResetPostgres {
       StatusDefinitions.Status.builder()
           .setStatusText("Rejected")
           .setLocalizedStatusText(LocalizedStrings.withDefaultValue("Rejected"))
-          .setEmailBodyText(Optional.of("Rejected email body"))
           .setLocalizedEmailBodyText(
               Optional.of(LocalizedStrings.withDefaultValue("Rejected email body")))
           .build();
 
+  private static final StatusDefinitions.Status WITH_TRANSLATIONS_STATUS =
```

Comment:
nit/opt: WITH_STATUS_TRANSLATIONS reads better to me.

Reply:
Done.